src/transformers/GovUKPayPaymentEventMessage.test.ts

```diff
@@ -41,8 +38,8 @@ describe('message formatter', () => {
 		expect(body).toHaveProperty('reproject_domain_object', true)
 		expect(body).toHaveProperty('service_id')
 		expect(body).toHaveProperty('live', true)
-		expect(body).toHaveProperty('event_details.boolean_value', true)
-		expect(body).toHaveProperty('event_details.numeric_value', 123)
+		expect(body).toHaveProperty('event_details.boolean_string_with_spaces_true', true)
+		expect(body).toHaveProperty('event_details.boolean_string_with_spaces_false', false)
```

Comment:
Why would boolean_string_with_spaces_true be present but not boolean_string_with_spaces_false?

Reply:
Because we have stirng matching to look for the strings true and false. Everything is a string coming into the class under test and we are looking at the value inside the string and then turning it into a boolean. Without the additional test we're only testing half the logic

Reply:
Oh I misunderstood. The test is saying 'event_details.boolean_string_with_spaces_false' has boolean value false, not false as in "the boolean_string_with_spaces_false property doesn't exist".